docs/conceptual/query-expressions-[fsharp].md

```diff
@@ -1329,7 +1321,7 @@ VALUES(15, 7, 3);
 
 The following code contains  the sample code that appears in this topic.
```

Comment:
The formatting in this code sample is a little wonky (lack of tabs in the query expressions, no newline between `open` and a type definition).  If you're up for it, that could also be addressed here.  This wouldn't be a blocker for getting the PR merged, though.